### PR TITLE
ST: Make Docker registry as a configurable parameter

### PR DESCRIPTION
**Pull request by serrss**
### Type of change
- Enhancement / new feature

### Description

1. Added the ability to configure Docker registry via `DOCKER_REGISTRY` variable. These changes will help us to fully manage the Docker images that are used in system tests. 
2. `Environment.java` was moved to test package to have the ability to get images variables in `TestUtils`. 
3. `Environment.java` was refactored to a simpler state. Removed singleton and getters. Why do we need it? 

### Checklist
- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by serrss** [comment]
@strimzi-ci run tests

File: .travis/build.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ echo "Docker push with registry $DOCKER_REGISTRY and org $DOCKER_ORG under tag $
 make docker_push
 
 OLD_DOCKER_ORG=$DOCKER_ORG
-export DOCKER_ORG="localhost:5000/strimzici"
+export DOCKER_ORG="strimzici"
```
Comment (scholzj) [inline .travis/build.sh]: You removed the `localhost:5000` which was AFAIK used by Travis for STs. I do not see it added anywhere. How did you addressed it?

Comment (serrss) [reply .travis/build.sh]: Thanks for the question, you can see that we define `DOCKER_REGISTRY="localhost:5000"` for Travis in build.sh file, link - https://github.com/strimzi/strimzi-kafka-operator/blob/9b1afa8b925cad7ba924098e82789378ad174413/.travis/build.sh#L48

**Comment by Frawless** [comment]
Basically changes in `Environment` looks fine, logs looks fine as well. I have just few nits.
1. Is this approach synchronized or not? 
2. I prefer move some methods from `TestUtils` to `StUtils` instead of move `Environment` to test package. Methods which need values from `Environment` are used only in STs. All methods, which are not used in ITs and unit test can be moved to ST package.

**Comment by serrss** [comment]
Thanks all for the comments. @Frawless I agree with moving methods to configure docker images to `StUtils`. This will be a more reasonable solution. As for synchronized, we set all environment variables via scripts before java execution, so we can have env variables in java as constants.

**Pull request merged**